modules/machineset-modifying.adoc

```diff
@@ -5,91 +5,144 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="machineset-modifying_{context}"]
-= Modifying a compute machine set
+= Modifying a compute machine set by using the CLI
 
-To make changes to a compute machine set, edit the `MachineSet` YAML. Then, remove all machines associated with the compute machine set by deleting each machine or scaling down the compute machine set to `0` replicas. Then, scale the replicas back to the desired number. Changes you make to a compute machine set do not affect existing machines.
+Changes that you make to a compute machine set do not affect existing machines.
+To create machines that reflect the updated configuration in the `MachineSet` custom resource (CR), you must scale the compute machine set.
```

Comment:
i wonder if this is clear enough, would it be worthwhile to state that only new machines created after the configuration change will take the updated configuration?

Reply:
Thanks - hopefully the new commit helps :crossed_fingers: